### DISCUSSION TITLE
Sketch or Figma file available?

### DISCUSSION THREAD

**Discussion by dsomar** [commercetools/ui-kit#2061] [Q&A]
Hi there,

Is there a source Sketch or Figma file available for the UI kit?

**Comment by FilPob** [comment]
Hi @dsomar Thanks for reaching out! We are actually considering releasing a public Sketch library for UI Kit. As we are just gathering informations about this, we would appreciate if you could specify your personal use case a bit. 

1. For which purpose do you need this library and how would a process look like, where you use this library during the development process?
2. What would you expect to find in this library?
3. Is there a specific reason why you would prefer Sketch or Figma?

If you would like to provide even more feedback, we would be glad to reach out to you and schedule a session for further questions.

**Reply by tbrasington** [reply]
Adding my vote here for a Figma kit. Main use case for me is that when developing custom pages and views for commecetools it would be great to use the same components that are in storybook.

And Figma because its cross-platform, and it could be a resource in the Community section.

**Reply by FilPob** [reply]
Thanks @tbrasington, noted this down as well.

**Reply by dsomar** [reply]
Am seconding my vote with @tbrasington - Figma would be ideal but Sketch can work too and I'd probably end up converting it myself.

We have the same core use case in that we are developing some custom admin pages and while I can make a mental leap that _a button is a button_, it's especially important to build handoff documents where the engineers don't have to do any additional design-to-code translation than the work that they're already doing.

I would hope and expect to find the same Storybook components in this library.

**Reply by FilPob** [reply]
Thanks @dsomar for specifying your use case. We will keep you guys posted about our progress.
